packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1429,6 +1428,26 @@ export class ContainerRuntime
 
 		ReportOpPerfTelemetry(this.context.clientId, this.deltaManager, this.logger);
 		BindBatchTracker(this, this.logger);
+
+		this.mc.logger.sendTelemetryEvent({
```

Comment:
If possible (and makes logical sense), we should consider reducing number of events we fire on open path and merging them